packages/debug/src/__tests__/basic.test.ts

```diff
@@ -1,15 +1,17 @@
 import stripAnsi from 'strip-ansi'
 
-import Debug, { getLogs } from '..'
+import { Debug, getLogs } from '../index'
 import { removeISODate, sanitizeTestLogs } from '../util'
 
 describe('debug', () => {
-  test('shouldnt log if its not enabled', () => {
+  test('should not log if it is not enabled', () => {
     const debug = Debug('my-namespace')
     const logs: string[] = []
+
     debug.log = (...args) => {
-      logs.push(stripAnsi(args[0]).trimStart())
+      logs.push(stripAnsi(`${args[0]}${args[1]}`).trim())
```

Comment:
Why do we need `args[1]` added now?

Reply:
So that's the standard way for hooking in a logger. Not sure why we were doing the concatenation ourselves. That's the way the api works, now it works in a more standard way.
```ts
    const debug = Debug('my-namespace')

    debug.log = (...args) => {
      logs.push(stripAnsi(`${args[0]}${args[1]}`).trim())
    }
```

 I can make that it behaves like the previous one if needed, but pretty much internal, so I did this choice.

Reply:
`...args` is supposed to be the raw arguments passed to a `debug` call, not a stringified version of it (like we had).

Reply:
(probably not used besides the test)

Reply:
Are you ok with that @Jolg42?

Reply:
We checked on a huddle 👍🏼